Atomic/syscontainers.py

```diff
@@ -227,6 +227,121 @@ def build_rpm(self, repo, name, image, values, destination):
         return None
 
     def install(self, image, name):
+        """
+        External container install logic.
```

Comment:
Right now there are 3 (or more?) places that document labels. I'm planning on adding it in the [Atomic System Containers repo](https://github.com/projectatomic/atomic-system-containers/blob/master/LABELS.md) and, as it is used, should propagate to wherever becomes the proper place. Or at least that is the theory.